pkg/schema/leaf.go

```diff
@@ -125,8 +125,14 @@ func toSchemaType(e *yang.Entry, yt *yang.YangType) (*sdcpb.SchemaLeafType, erro
 		}
 	case yang.Yleafref:
 		var err error
-		leafSchemEntry := e.Find(yt.Path)
-		slt.LeafrefTargetType, err = toSchemaType(leafSchemEntry, leafSchemEntry.Type)
+		normalizedPath := normalizePath(yt.Path, e)
+		leafSchemaEntry, err := sc.GetEntry(normalizedPath)
+		if err != nil {
+			return nil, err
+		}
+		lsePath := leafSchemaEntry.Path()
```

Comment:
@alexandernorth this is unnecessary code, not?

Reply:
Yes you were right sorry this is left over from testing - removed now